Sources/RichTextKit/Actions/RichTextAction.swift

```diff
@@ -77,6 +77,8 @@ public enum RichTextAction: Identifiable, Equatable, RichTextLabelValue {
 
     /// Undo the latest change.
     case undoLatestChange
+
+    case link(url: URL?)
```

Comment:
I'd prefer if we keep the alphabetical order for this enum and all switches over it.